app/routes/_gcn.circulars._archive._index/SortSelectorButton.tsx

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { useSubmit } from '@remix-run/react'
+import {
+  Button,
+  ButtonGroup,
+  CardBody,
+  Grid,
+  Icon,
+  Radio,
+} from '@trussworks/react-uswds'
+import { useState } from 'react'
+
+import DetailsDropdownContent from '~/components/DetailsDropdownContent'
+
+function SortButton({
+  sort,
+  ...props
+}: {
+  sort?: string
+} & Omit<Parameters<typeof ButtonGroup>[0], 'segmented' | 'children'>) {
+  const slimClasses = 'height-4 padding-y-0'
+
+  return (
+    <ButtonGroup type="segmented" {...props}>
+      <Button type="button" className={`${slimClasses} padding-x-2`}>
+        Sort by...
+      </Button>
+      <Button type="button" className={`${slimClasses} padding-x-2`}>
+        {<Icon.FilterList role="presentation" />}
+      </Button>
+    </ButtonGroup>
+  )
+}
+
+export function SortSelector({ sort = 'circularId' }: { sort?: string }) {
```

Comment:
```suggestion
export function SortSelector({ sort = 'circularId' }: { sort?: string }) {
```

How about `defaultValue` instead of `sort`?

And define the type as:

```ts
type SortOrder = 'circularId' | 'relevance'

...

function SortSelector({value}: {value?: SortOrder})`
```